components/com_contact/tmpl/contact/default.php

```diff
@@ -121,7 +130,7 @@
 				<?php if ($tparams->get('allow_vcard')) : ?>
 					<?php echo Text::_('COM_CONTACT_DOWNLOAD_INFORMATION_AS'); ?>
 					<a href="<?php echo Route::_('index.php?option=com_contact&amp;view=contact&amp;id=' . $this->item->id . '&amp;format=vcf'); ?>">
-					<?php echo Text::_('COM_CONTACT_VCARD'); ?></a>
+						<?php echo Text::_('COM_CONTACT_VCARD'); ?></a>
```

Comment:
```suggestion
					<?php echo Text::_('COM_CONTACT_VCARD'); ?></a>
```
Revert wrong change on indentation.